tests/README.md

```diff
@@ -9,8 +9,8 @@ To setup a local development environment, install both local and test dependenci
 
 ```bash
 # clone the repo
-git clone https://github.com/PyTorchLightning/pytorch-lightning
-cd pytorch-lightning
+git clone https://github.com/Lightning-AI/lightning.git
+cd lightning
 
 # install required depedencies
 python -m pip install ".[dev, examples]"
```

Comment:
@Borda these need to be updated too, right?

Reply:
well yes, and atm we do not have joined extras if you install all, so if you develop Pl you shall run (and I would emphasise to install in developer mode `-e`)
```
export PACKAGE_NAME=pytorch
pip install -e ".[dev, examples]"
```
https://github.com/Lightning-AI/lightning/blob/c71f32a490dcbfea20882cf2e9d8b1439c492142/setup.py#L27

Reply:
@Borda

Reply:
```suggestion
export PACKAGE_NAME=pytorch
python -m pip install ".[dev, examples]"
```